### DISCUSSION TITLE
Readings in prep for week 12

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#14] [Reading responses etc]
After reading Paul Ford's [tour of circuits building into algorithms](https://www.bloomberg.com/graphics/2015-paul-ford-what-is-code/#lets-begin), Crawford and Joler's [expedition through the Amazon Echo](https://www.anatomyof.ai)'s material and human networks, and Gries et al's more theoretical exploration of [objects and agency in the digital humanities](https://canvas.pitt.edu/courses/220016/files?preview=14626125), **please write a short response to let us know where your mind went**. For example, you might...

* Highlight new ideas or key terms, especially those that you expect will be useful in thinking about digital projects.
* Highlight sentences, passages, or methods/approaches that struck you as especially well done, perhaps worth imitating.
* Link outward to something a passage reminds you of, explaining where you're leading us.
* Pose questions to your classmates about a passage, especially (but not exclusively) if something is confusing.
* Respond to any of the above – which you can now do directly, using threads.

Where possible, please try to point us to at least one particular passage or chunk from the text you're responding to – quoting if you can, giving a timestamp or page reference either way – as a way to help keep our discussion grounded.

**Comment by SidraNArshad** [comment]
Mark Graham is quoted in "Anatomy of an AI System" saying that, "contemporary capitalism conceals the histories and geographies of most commodities in the here and now of time and space, and rarely have the opportunities to gaze backwards through the chains of production in order to gain knowledge about the sites of production, transformation, and distribution." This quote, and really the entirety of Crawford and Joler’s piece on the obfuscation of unethical labor and resource extraction practices that produce such oxymoronic concepts as the comforts provided by the Amazon Echo, reminded me of a semi-viral tumblr post that comes back every once and a while to keep me up at night and lead me to question my own place in the chain of actions and events discussed in the reading. I’ve included a screenshot of the post—but what the post is most memorable for is its lasting reminder to conceive of the people under the floors, the ones who were there in the past and the ones who are there right now. I think that the discussion of these people made invisible in the eyes of the general, Western public is especially important in our current moment as the ongoing conflict  in Congo has been exacerbated by the increased mining of cobalt and copper. The reality of this moment is that the people who have been historically hidden away under the floors are right in front of our eyes, and yet they continue to go unseen. 


![The people under the floors ](https://github.com/benmiller314/dsam2023fall/assets/142813292/a83afd03-fd04-4b8d-95e9-4c8bf2578570)

**Comment by amj169** [comment]
The “Anatomy of an AI System: The Amazon Echo As An Anatomical Map of Human Labor, Data and Planetary Resources” reminded me of the Ring doorbell incident with the LAPD in 2016- 2018 (ish). Basically, Amazon silently partnered with LAPD in some backwards, fucked up race driven MLM scheme as an excuse to patrol and profile people without a warrant by collecting Ring doorbell footage from people they sold them to. The ring takes on extremely problematic labor roles for the police. It’s a tool for data collection and video footage that they normally wouldn’t have in their arsenal that they now have access to, and this “extra” footage put marginalized people in their own communities at risk in new ways than before. If you were visiting a place where you were being publicly recorded like a shopping center and were just walking by, nobody was going to pull the footage and call the police for you literally just existing. But in some reported LAPD-Ring cases, home owners would call the police if they saw a suspicious person walk up their street (of course they were always black or brown). This adds unnecessary police confrontations with people who were quite literally just walking from one place to another in their own neighborhoods and allows for additional discrimination through real time racial profiling. 

Here's an article that covers this briefly https://www.eff.org/deeplinks/2021/06/emails-show-amazon-rings-hold-lapd-through-camera-giveaways

One final note on labor that I’ll keep brief so we can talk about it in class-I wonder how ai will change academic labor when it comes to the publishing industry and tenure track positions. How are journals treating ai usage in terms of authorship? What does this mean for the H-index? Ai as a tool is just exposing existing inequitable practices. I just hope this might mean there is a "new" opportunity to do something about those systems. 

Also--I have to say that I started formulating some of my response before I got the the part where they talk about the mechanical Turk. I was going to mention that the Statua citofonica reminded me of the mechanical Turk! It's still W I LD that that happened about 200 years before computer chess as we know it was possible.

**Comment by chloe-cmu** [comment]
The readings that focused on the physicality of data highlighted a concept that has come up a couple of times for me in this class, which is that though it’s easy to think of the digital as an entirely separate world, but that idea doesn’t actually work all that well. Digital tools are incredibly powerful, but that power comes from a difference in speed and scale, rather than kind. (Ford points to this: "You, using a pen and paper, can do anything a computer can; you just can’t do those things billions of times per second.") Crawford and Joler’s piece highlights something similar; that the networks that make an Amazon Echo possible are not fundamentally new, but rather more densely layered than previous supply chains (Crawford and Joler, XII). As a historian interested in capitalism, reading about the physicality of data emphasized that digital inequities are often the same old inequities, just gussied up in different clothing (this was something most explicitly talked about in the New Jim Code, I believe.) Analytically, that to me suggests that we don’t really need to come up with new paradigms as scholars, but can make significant headway in analyzing digital projects using the techniques we already have.

**Comment by Rezaetmd** [comment]
I found Kate Crawford and Vladan Joler’s article interesting as it discusses the quantification of nature and human experiences through AI systems and explains how this leads to the privatization of knowledge and resources by encroaching on the commons and converting them into commodities. They liken his process to the historical enclosure of land and forests, as well as the privatization of water resources. The article argues that the same process is now occurring with machine learning, as the new gold rush in the context of artificial intelligence is to enclose different fields of human knowing, feeling, and action in order to capture and privatize those fields. This privatization is exemplified by the extraction of knowledge value from public data, where the meta-value of the data is privately owned. They argue that this leads to a future where expert local human labor in the public system is increased and sometimes replaced with centralized, privately-owned corporate AI systems, using public data to generate enormous wealth for a select few.

**Comment by MustafaKandil** [comment]
This week's readings highlight a shift in how we approach humanities research, encouraging a more inclusive and balanced perspective that moves away from old-fashioned ways of thinking. The idea is to consider both humans and non-human elements, especially in the context of digital humanities. The authors suggest viewing digital technologies as active participants in research, stressing the importance of ethical considerations and exploring the unique aspects of these tools.

In a different piece about the Amazon Echo, the authors metaphorically break down the device's workings, revealing the hidden aspects of its creation and impact. They talk about the people involved in making it, the data it collects, and the environmental consequences. This essay prompts us to think more deeply about the social and environmental impacts of everyday technologies.

Together, these readings encourage us to look at technology and research in a more comprehensive way, considering not just their immediate effects but also their broader implications for society, the environment, and our understanding of different fields.
